RealEstateCore.DTDLv2.jsonld

```diff
@@ -695,6 +705,25 @@
         },
         "writable": true
       },
+      {
```

Comment:
This looks like it's a simple map `(string->string)` instead of the specified `(string->(string->string))`

```
{
  "@type": "Property",
  "displayName": {
    "en": "Identifiers"
  },
  "name": "identifiers",
  "schema": {
    "@type": "Map",
    "mapKey": {
      "name": "namespace",
      "schema": "string"
    },
    "mapValue": {
      "name": "identifier",
      "schema": {
        "@type": "Map",
        "mapKey": {
          "name": "key",
          "schema": "string"
        },
        "mapValue": {
          "name": "value",
          "schema": "string"
        }
      }
    }
  },
  "writable": true
},
```